.project-keeper.yml

```diff
@@ -21,14 +19,6 @@ build:
             run: |
               mvn --batch-mode install -DskipTests
               mvn --batch-mode com.exasol:quality-summarizer-maven-plugin:summarize
-
-    - name: "release.yml"
-      stepCustomizations:
-        - action: INSERT_AFTER
-          stepId: build
-          content:
-            name: Run summarizer mojo
-            id: maven-run-summarizer
-            run: |
-              mvn --batch-mode install -DskipTests
-              mvn --batch-mode com.exasol:quality-summarizer-maven-plugin:summarize
```

Comment:
Tests don't run in the release build, so Jacoco doesn't run and the plugin won't find `jacoco.xml`.